QuestionAnswering/QuestionAnswering/InferenceModule.mm

```diff
@@ -57,10 +57,13 @@ - (int)argMax:(NSArray*)array {
             inputs[i] = [tokenIds[i] longValue];
         at::Tensor tensorInputs = torch::from_blob((void*)inputs, {1, MODEL_INPUT_LENGTH}, at::kLong);
         
-        torch::autograd::AutoGradMode guard(false);
```

Comment:
Why has this been removed?

Reply:
Once we switched to lite interpreter, we don't need this guard anymore.